src/Engine/GameAction.java

```diff
@@ -32,19 +32,13 @@
  */
 public interface GameAction
 {
-  public enum ActionType
-  {
-    UNIT, PRODUCTION, OTHER
-  }
-
   /**
    * Returns a GameEventQueue with the events that make up this action. If the action
    * was constructed incorrectly, this should return an empty GameEventQueue.
    */
   public abstract GameEventQueue getEvents(MapMaster map);
   public abstract XYCoord getMoveLocation();
   public abstract XYCoord getTargetLocation();
-  public abstract ActionType getType();
   public abstract UnitActionType getUnitActionType();
```

Comment:
Huh. I totally figured it would be used. Maybe rename `getUnitActionType()` to avoid verbosity? Then again, maybe we should delay a change like that to avoid merge conflicts for now.

Reply:
I've never been too worried about verbosity, though I'm fine with changing it. I'm also fine with deferring that change to avoid conflicts.